.deployment.config.json

```diff
@@ -81,6 +81,16 @@
         }
       }
     },
+    {
+      "id": "invalidate_cache",
+      "team_jenkins": {
+        "job_name": "ui-kit-cloudfront-invalidation",
+        "disabled": true,
+        "prd": {
+          "disabled": false
+        }
+      }
```

Comment:
It feels like we could use `ui-kit-production-release` job below?

Reply:
So merge both tasks together? could be doable, i'll see if the jobs are compatible and do this

Good proposition!